example/index.htm

```diff
@@ -133,6 +151,7 @@
     time_anim: false,
     gyroscope: true,
     webgl: true,
+    usexmpdata: false,
```

Comment:
I don't see the relation between disabling XMP parsing and the preload, and there shouln't be any.